development/src/phone-hooks/metadata/countries.json

```diff
@@ -2121,7 +2121,13 @@
     "ae",
     "United Arab Emirates",
     "971",
-    "+971 (..) ... ... ...."
+    "+971 (.) ... ...."
```

Comment:
```suggestion
    "+971 (.) ... ... ....."
```

Reply:
This might work for toll free number starting with 800 but for the normal numbers is quite off.